include/baresip.h

```diff
@@ -129,6 +129,8 @@ const char *account_call_transfer(const struct account *acc);
 const char *account_extra(const struct account *acc);
 int account_uri_complete(const struct account *acc, struct mbuf *buf,
 			 const char *uri);
+int account_adelay(const struct account *acc);
```

Comment:
it is difficult to know what _a_ in _adelay_ means, without knowing the whole context.
perhaps better to write the full name, e.g. "answerdelay"

Reply:
Yes, I will change this to:
```
int account_answerdelay(const struct account *acc);
void account_set_answerdelay(struct account *acc, int adelay);
```
Because this are API functions.

The field name in `struct account` should stay short: `adelay`